tests/Models/Employee.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Spatie\Activitylog\Test\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Spatie\Activitylog\Traits\CausesActivity;
+use Illuminate\Contracts\Auth\Authenticatable;
+
+class Employee extends Model implements Authenticatable
```

Comment:
Seems to me that the code is very similar to the `User` testmodel. Can you extends that? Or create a `BaseUser` class and let both `User` and  `Employee` extend that?

Also I don't like the name `Employee`. This model should be named after something you're needing it for to test. `OtherGuardUser` seems better (maybe there's an even better name, I'm open for suggestions).